Formula/o/orogene.rb

```diff
@@ -0,0 +1,32 @@
+class Orogene < Formula
+  desc "`node_modules/` package manager and utility toolkit"
+  homepage "https://orogene.dev"
+  url "https://github.com/orogene/orogene/archive/refs/tags/v0.3.31.tar.gz"
+  version "0.3.31"
+  sha256 "06105fde1116958e97db8be1ccf4734bcecd8d4282e6c6c27ae9929fa51a2ac8"
+  license "Apache-2.0"
+  head "https://github.com/orogene/orogene.git", branch: "main"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, arm64_sonoma:   "d76286dbec347fd26739d5e19742c54b30fc7616a780e499812db36a2c3b8102"
+    sha256 cellar: :any_skip_relocation, arm64_ventura:  "e30c0cf05dbcb16a4a5820126dd105067974e3a82d786a07fd0e5b2f36fb8085"
+    sha256 cellar: :any_skip_relocation, arm64_monterey: "e30c0cf05dbcb16a4a5820126dd105067974e3a82d786a07fd0e5b2f36fb8085"
+    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "e30c0cf05dbcb16a4a5820126dd105067974e3a82d786a07fd0e5b2f36fb8085"
+    sha256 cellar: :any_skip_relocation, sonoma:         "f00c87a3ff4f645c600903f0ea7861f3629a10559b60ca9215e71135d427554c"
+    sha256 cellar: :any_skip_relocation, ventura:        "14919101e5f7d51d091463a83ab2adad69e808991fc6e957ff5c6b7e46f9d85c"
+    sha256 cellar: :any_skip_relocation, monterey:       "14919101e5f7d51d091463a83ab2adad69e808991fc6e957ff5c6b7e46f9d85c"
+    sha256 cellar: :any_skip_relocation, big_sur:        "14919101e5f7d51d091463a83ab2adad69e808991fc6e957ff5c6b7e46f9d85c"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:   "d0c8001ffac28e44610535a60fd5a3bca5b649850f2360a2fec2676f7b2255a0"
+  end
+
+  depends_on "rust" => :build
+  depends_on "openssl"
```

Comment:
```suggestion
  depends_on "openssl@3"
```

Reply:
I believe this should be @1

Reply:
OpenSSL 1.1.1 is end of life. We will not accept new formulae that require it.